ballerina/tests/request_response_tests.bal

```diff
@@ -0,0 +1,265 @@
+// Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/log;
+import ballerina/lang.runtime;
+import ballerina/test;
+import ballerina/uuid;
+
+final string[] completedTokens = [];
+final string[] receivedErrors = [];
+
+final Service reqResService = service object {
+    remote function onMessage(Message message, Caller caller) returns error? {
+        log:printInfo(check string:fromBytes(message.payload));
+        Error? err = caller->respond({
+            payload: ("Response for " + check string:fromBytes(message.payload)).toBytes()
+        });
+        if err is Error {
+            receivedErrors.push(err.message());
+        }
+    }
+
+    remote function onError(Error err) returns error? {
+        log:printError("Error occured ", err);
+        receivedErrors.push(err.message());
+    }
+
+    remote function onComplete(DeliveryToken token) returns error? {
+        log:printInfo("Message delivered " + token.messageId.toString());
+        completedTokens.push(token.topic);
+    }
+};
+
+@test:Config {enable: true}
```

Comment:
```suggestion
@test:Config {}
```

Reply:
Using this to easily disable tests and run only a single test